tb/data/concat_config.h

```diff
@@ -0,0 +1,23 @@
+#include <ap_int.h>
+
+#define NUM_INPUTS 4
```

Comment:
Sorry if I was not quite clear. There's nothing special about `SIMD`. Please, replace al of these `#define`s by a typed `constexpr`.